eclass/latex-package.eclass

```diff
@@ -105,79 +94,73 @@ latex-package_src_doinstall() {
 
 	# This actually follows the directions for a "single-user" system
 	# at http://www.ctan.org/installationadvice/ modified for gentoo.
-	[ -z "$1" ] && latex-package_src_install all
+	[[ -z ${1} ]] && latex-package_src_install all
 
-	while [ "$1" ]; do
-		case $1 in
+	while [[ ${1} ]]; do
+		case ${1} in
 			"sh")
-				for i in `find . -maxdepth 1 -type f -name "*.${1}"`
+				find . -maxdepth 1 -type f -name "*.${1}" | while read i;
```

Comment:
Don't do pipelines like that, ebuild code gets executed in subshell instead of main process. You have to do the `while ...; do ...; done < <(find ...)`; also print0 etc., find it in other eclasses.

But here I think you can just do `find ... -exec dobin {} +` as `dobin` as `these commands must all be external programs and not bash functions or aliases`.